src/config.ts

```diff
@@ -40,9 +40,17 @@ export default {
         'You can assign this role to yourself to subscribe to get open-source reminders and if you like contributing to open-source software',
     },
   },
-  defaultEmbed: () => {
+  COLORS: {
```

Comment:
I'm not sure if this should be all caps.

Reply:
We could change to `colors` or `col`.
I would like colors as it is more explicit.

Share your thoughts and I will change it.

Reply:
I think colors would be much easier to alter in the future, for contributors

Reply:
agreed, `colors` is preferable 👍

Reply:
Alright I will change it tomorrow morning 👍